cmd/veneur/main.go

```diff
@@ -94,38 +100,60 @@ func main() {
 
 	packetPool := &sync.Pool{
 		New: func() interface{} {
-			return make([]byte, veneur.Config.MetricMaxLength)
+			return packet{
+				buf: make([]byte, veneur.Config.MetricMaxLength),
+				ip:  "",
+			}
 		},
 	}
 
+	veneurIPdigester := fnv.New32()
```

Comment:
I must admit this confused me or longer than I'm happy with. :) Can you add a comment here describing that you're pre-computing the digest of our "built in" metric such that we don't have to do so in each iteration?